Build fails if only curl-backend feature is on

**Problem**
```
nabijaczleweli@tarta:~/uwu/rustup$ cargo build --release --no-default-features --features curl-backend
   Compiling syn v1.0.56
   Compiling itoa v0.4.7
   Compiling rustup v1.23.1 (/home/nabijaczleweli/uwu/rustup)
   Compiling download v0.6.8 (/home/nabijaczleweli/uwu/rustup/download)
error[E0061]: this function takes 3 arguments but 4 arguments were supplied
   --> download/src/lib.rs:39:34
    |
39  |         Backend::Reqwest(tls) => reqwest_be::download(url, resume_from, callback, tls),
    |                                  ^^^^^^^^^^^^^^^^^^^^ ---  -----------  --------  --- supplied 4 arguments
    |                                  |
    |                                  expected 3 arguments
    |
note: function defined here
   --> download/src/lib.rs:439:12
    |
439 |     pub fn download(
    |            ^^^^^^^^
440 |         _url: &Url,
    |         ----------
441 |         _resume_from: u64,
    |         -----------------
442 |         _callback: &dyn Fn(Event<'_>) -> Result<()>,
    |         -------------------------------------------

error: aborting due to previous error

For more information about this error, try `rustc --explain E0061`.
error: could not compile `download`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: build failed
```
**Steps**
1. Be at 3cca9ff11dd4ea31f2611c03cb5861baa1a13a9c
2. `cargo build --release --no-default-features --features curl-backend`

**Possible Solution(s)**
Removing the `tls` argument worked, but I'm assuming it's somewhat important, so.

**Notes**

Output of `rustup --version`: `rustup 1.23.1 (3df2264a9 2020-11-30)`
Output of `rustup show`:
```
nabijaczleweli@tarta:~/uwu/rustup$ rustup show
Default host: x86_64-unknown-linux-gnu
rustup home:  /home/nabijaczleweli/.rustup

installed toolchains
--------------------

stable-x86_64-unknown-linux-gnu
nightly-x86_64-unknown-linux-gnu
1.35.0-x86_64-unknown-linux-gnu

installed targets for active toolchain
--------------------------------------

x86_64-unknown-linux-gnu
x86_64-unknown-linux-gnux32
x86_64-unknown-linux-musl

active toolchain
----------------

stable-x86_64-unknown-linux-gnu (default)
rustc 1.49.0 (e1884a8e3 2020-12-29)
```

--- a/download/src/lib.rs
+++ b/download/src/lib.rs
@@ -433,6 +433,7 @@ pub mod curl {
 pub mod reqwest_be {
 
     use super::Event;
+    use super::TlsBackend;
     use crate::errors::*;
     use url::Url;
 
@@ -440,6 +441,7 @@ pub mod reqwest_be {
         _url: &Url,
         _resume_from: u64,
         _callback: &dyn Fn(Event<'_>) -> Result<()>,
+        _tls: TlsBackend,
     ) -> Result<()> {
         Err(ErrorKind::BackendUnavailable("reqwest").into())
     }

